pubspec.yaml

```diff
@@ -18,18 +18,17 @@ dependencies:
     sdk: flutter
   http: "<1.2.0"
   shared_preferences: ">=2.0.0 <3.0.0"
-  mutex: ^3.0.0
-  dio: ^4.0.0
+  mutex: ^3.1.0
+  dio: ^5.3.3
```

Comment:
can we keep this >= 4.0.0 and  < 6.0.0? Cause otherwise this will be a breaking change for users who use an older version (4.x.x) of dio

Reply:
I think no, otherwise we will lose users who want to use the latest version of dio (in my case for example latest sentry sdk for flutter uses dio > 5), so yes, this is change that needs to be made. Old users may easy update his dio or stay on outdated version

Reply:
>  otherwise we will lose users who want to use the latest version of dio

Specifying a version of >= 4.0.0, < 6.0.0 should include the latest version, correct? So why would we lose those users? Am i missing something?

Reply:
Correct, however in dio since 5 version some class names were changed. And if we install version dio 4, we will get errors related to non-existent classes. So including version 4 don't make sense

Reply:
Right. I see. Okay. So this will be a breaking change from our SDK. Could you please bump the version to `0.3.0`?

Reply:
Ok